resources/email.py

```diff
@@ -17,8 +17,6 @@ def post(self):
         if not data.userid or not data.title or not data.body:
             return {'Message': 'Bad Request'}, 400
 
-        #TODO add feature to send to multiple emails
```

Comment:
We don't need to worry about batching emails yet, at least not in this version. I was thinking we would setup an email notification system at some point. The idea being that that the backend would periodically audit the existing tickets and dispatch emails based on how long they hadn't been reviewed, changes, other flags. In this use case, we would need to be able to send a batch of messages.